third_party/accessibility/ax/platform/ax_platform_node_mac.mm

```diff
@@ -699,6 +710,10 @@ - (NSValue*)AXSelectedTextRangeInternal {
     start = _node->GetIntAttribute(ax::mojom::IntAttribute::kTextSelStart);
     end = _node->GetIntAttribute(ax::mojom::IntAttribute::kTextSelEnd);
   }
+  if (start == -1 && end == -1) {
+    return [NSValue valueWithRange:{0, 0}];
```

Comment:
Is the range here expected to be a zero-length range at position 0? AppKit APIS typically use `NSMakeRange(NSNotFound, 0)`.